### DISCUSSION TITLE
Channel in Phoenix app goes offline

### DISCUSSION THREAD

**Discussion by Darth-Coin** [ACINQ/phoenix#315] [Q&A] [answered]
Hi, 
I am using latest version 1.4.23 (38) on Android 8.1 device, clearnet. Doing some [tests with many different LN wallet apps](https://github.com/hsjoberg/blixt-wallet/issues/1065).
After receiving some amounts, I end up with 3 channels and one is always staying offline. See image attached.
I tried restarting, I tried clicking on the offline channel details but is not showing anything.
All other channels are functional, I can make payments with them.
Is there a procedure to bring back online that channel?
Phoenix wallet nodeID is: 0285c45fa806713e86171d0fd9a3043fbde9cae0a36258d7d2a5967187da56324a
This is a testing wallet, so I don't mind posting public info. I just want to finish my tests.
I didn't create an issue, because I am not sure if is a bug or just a glitch or something. So first I ask how to fix this by myself if is possible.

![image](https://user-images.githubusercontent.com/56997124/203022877-f16d7977-e8f6-4bfc-a69a-849adf1fb9b3.png)

**Comment by pm47** [comment]
I can confirm this is a display glitch, you only have 2 channels. That "offline" channel is a channel which failed at opening due to a disconnection, and was replaced by a new one (the last in the list).

In eclair, this would be a case of `BITCOIN_FUNDING_TIMEOUT` and channel would be cleaned up after two weeks, but I'm not quite sure how we deal with this in lightning-kmp.

**Comment by Darth-Coin** [comment, marked as answer]
OK, I understand now, I was just worried that those 12k sats are gone or something, but are actually replaced in another channel.
So I just have to wait 2 weeks to see what happen, refreshing everyday that channel list? I can do that, no problem.
Meanwhile I can just use the app as usual right?

How Phoenix is deciding to open channels ? Is there a calculation based on the amount is received?
For example:
- if I send 50k sats will open a channel of 50k+ 10%
- so a next payment of 4500 sats I could just receive in the same channel?

Or is opening a channel for each received payment?
I am asking to understand better how Phoenix app is dealing with liquidity so I can use it better.
